### ISSUE TITLE
Disable (or make it a buried feature) the ability to set more than one row/column of table headers.

### ISSUE THREAD

**Issue by nikkimk** [ckeditor/ckeditor5#5584]
## 📝 Provide a description of the new feature
A11y feature: Right now the UI makes it easy to create complex tables with more than one row/column of headers, which can be problematic for some screen readers. In fact, a whole table can be set as a header with the current UI. (See https://webaim.org/techniques/tables/data#complextables)

*What is the expected behavior of the proposed feature?*
Header toggle should only allow the first row/column of headers. (Anything more, since it compromises a11y should be hidden deeper into the UI.

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Reinmar** [comment]
Thanks for reporting this idea. I wonder if people actually use the ability to set multiple rows/cols as headers. I'd say most of them don't use any headers at all. Unfortunately, we cannot collect usage statistics in any way :(

And as for the accessibility aspect... cc @Comandeer

**Comment by mgifford** [comment]
As part of the We4Authors Cluster we're looking at improving the authoring tools https://accessibilitycluster.com/authoringTool/

Being able to create accessible tables is an important part of this.  Not everyone will use this correctly, but that's true of any tool. 

One approach that is being considered is just that of using templates like 
<img width="1042" alt="Screen Shot of We4Authors Example" src="https://user-images.githubusercontent.com/116832/94723001-1885e200-0326-11eb-8eb4-b48733d333d7.png">

Certainly having some visual means to differentiate headers would be important. Right now there is no means to do that.  

With being able to merge cells, it is also important to be able to assign which header is associated with that cell.

**Comment by mgifford** [comment]
I really think simplifying the Header tool really needs to be considered here. In Drupal the top row is assumed to be the header. I believe in CKEditor 5 Each header cell needs to be specified as a header individually, and that without any input there would be no Table Headers. This doesn't seem to be the right pattern.  There is a lot of merit to the approach above, but also of simply defaulting to the 1st row being headers (which can be overridden).  

@nikkimk I kinda feel that the current process of identifying headers is pretty buried are rarely used.  We could put a warning on this perhaps.  I really don't know why there doesn't seem to be an accessibility page for CKEditor that would highlight how to do complex tables correctly. Ideally we'd just link to that. 

https://www.funka.com/en/projekt/cluster-feature-testing/

**Comment by jakegibs617** [comment]
There are lots of examples where complex multi-level headers are needed. And we cannot do this currently with CKEditor5

https://www.w3.org/WAI/tutorials/tables/multi-level/ "This page covers tables that have multi-level header cells associated per data cell. Such tables are too complex to identify a strict horizontal or vertical association between header and data cells. In such tables, each table header is represented by a (document-wide) unique id. Data cells refer to those ids by listing one or more in their headers attribute, separated by spaces."
